cluster/config-defaults.yaml

```diff
@@ -1,3 +1,6 @@
+# temporary feature toggle for the "kubernetes" endpoint reconciler, default is "lease", otherwise "none"
+endpoint_reconciliation: "true"
```

Comment:
You need two feature toggles (one to rollout the updated service, another one to disable the reconciler). This will disable the reconciler first but rollout the service later.